lib/modules/play/widgets/node_display.dart

```diff
@@ -23,7 +23,7 @@ class NodeDisplay extends StatelessWidget {
     final play = context.watch<PlayScreenState>();
     final audioPlayer = play.audio;
     final actor = play.actors[node.actorId];
-    final text = play.tr.node(node, allowEmpty: true);
```

Comment:
_[issue]_ Sometimes nodes need to have no text. For example, when a node used as cell-writer or as a logical route. Or even media-only nodes that just show image \ audio. For such nodes no message card should be displayed. Without `allowEmpty`, we have no way to know when a node is not indented to have any text. 

Alternatively, we might introduce `bool display` field for `Node` model, with corresponding toggle in the Node Editor. This will be additionally useful for the timer management, as there's no need to wait and show the typing indicator for such "logic" nodes.

Or we can take it even further, and make an array of new `Content` objects for `Node`. The `Content` class will have various descendants such as `TextContent`, `AudioContent`, `ImageContent`, etc., and editors will be able to array them in the editor, controlling the contents of a node. And so a "logic" node will be that with empty `contents` array.

Reply:
An example of logic node from the tutorial:
<img width="449" alt="image" src="https://user-images.githubusercontent.com/32439229/168422281-76c831b8-8b8c-4b12-ba3f-fa20825a8f2c.png">